src/commands/validate_event.yml

```diff
@@ -0,0 +1,56 @@
+description: >
+  Validates that an event succeeded by polling the Sym Events API until we receive an event status of success or error.
+  The command will fail if an error status is received.
+  The environment variable SYM_JWT must be set for this orb to work.
+  For instructions on how to generate a Sym Token, see: https://docs.symops.com/docs/using-bot-tokens
+parameters:
+  event_id:
+    type: string
+    description: |
+      The ID of the Run to de-escalate. One of event_id or request_slug must be defined.
```

Comment:
?
```suggestion
      The ID of the event to validate. One of event_id or request_slug must be defined.
```

Reply:
whoops good catch. Missed that description update